000-index.md

```diff
@@ -1,5 +1,9 @@
 # Libuv Enhancement Proposals
 
+## Draft
```

Comment:
I don't think we need to land this as draft, we can probably go straight for aproval or rejection.